example/src/DrawerExample.re

```diff
@@ -0,0 +1,29 @@
+open BsReactNavigation;
+open Config;
+
+module Drawer =
+  DrawerNavigation.Create({
+    type item = Config.item;
+
+    let items = [Dashbord, Settings];
+    let drawerOptions =
+      DrawerNavigation.drawerOptions(~activeTintColor="#847", ());
+
+    let order = [Dashbord, Settings];
+
+    let getItem = tab =>
+      switch (tab) {
+      | Dashbord => (
+          "Dashbord",
+          (() => <Items.Dashboard />),
```

Comment:
I don't think this should be a function. I would suggest we keep it similar to Stack.